src/test/java/io/confluent/examples/streams/IntegrationTestUtils.java

```diff
@@ -394,4 +427,55 @@ public V setValue(final V value) {
     return result;
   }
 
+  /**
+   * A Serializer/Deserializer/Serde implementation for use when you know the data is always null
+   * @param <T> The type of the stream (you can parameterize this with any type,
+   *           since we throw an exception if you attempt to use it with non-null data)
+   */
+  static class NothingSerde<T> implements Serializer<T>, Deserializer<T>, Serde<T> {
```

Comment:
Added this because I didn't like seeing us put in a random serde, knowing that it wouldn't actually be used.